docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+   app:
+       build:
+           context: .
+       ports:
+           - "8001:8000"
```

Comment:
The application is exposed on port `8001` externally and mapped to `8000` internally. Ensure that this port configuration aligns with your deployment and testing requirements.